services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -20,6 +21,8 @@ export const AppRoutes = () => {
         <Route path="/standard" element={<ReportPageWrapper />} />
         <Route path="/reviewSubmit" element={<ReviewSubmitPage />} />
         <Route path="*" element={<NotFoundPage />} />
+        {/* MFP ROUTES */}
+        <Route path="/wp" element={<DashboardPage reportType="WP" />} />
```

Comment:
```suggestion
        <Route path="/wp" element={<DashboardPage reportType={ReportType.WP} />} />
```

and import the ReportType types?